ocs_ci/ocs/cluster.py

```diff
@@ -1631,6 +1631,29 @@ def get_mon_config_value(key):
     return mon_config_value
 
 
+def get_mds_config_value():
```

Comment:
I think it is better if this function returns config values for a particular key ( ie: mds cache memory limit) instead of default dumped values. Try: $ceph config show mds.<name>  mds_cache_memory_limit

Reply:
done